docs/reference/runtime/Execution.md

```diff
@@ -1,5 +1,9 @@
 # Execution
 
+Embedded systems are systems that are built for performing a sole dedicated duty. They are typically constrained in resources and contain only the necessary components to perform their duty. Most embedded systems are powered by a single microcontroller chip. That microcontroller is typically also limited in resources such as memory. Software for an embedded system must be carefully designed to make the best use of such limited resources. Mbed OS is often too large for such systems in its default configuration, but it supports additional modes that might be a better option depending on your target system.
```

Comment:
Regarding "...supports additional modes that...".
There are only two modes "Full profile" and "Bare metal profile". Since the first part of the sentence talks about the full profile, there is only one additional mode.
Should the sentence perhaps say:
"...supports a mode that..."

Reply:
Thanks for the review, now fixed.